src/common/components/header/index.js

```diff
@@ -2,73 +2,99 @@ import React, { Component, PropTypes } from 'react';
-                    <a>Hi, {user.name || user.login}</a>
-                  </li>
-                  <li className={ style['p-navigation__link'] } role="menuitem">
-                    <a href="/auth/logout" onClick={ this.onLogoutClick.bind(this)}>
-                      Sign out
-                    </a>
-                  </li>
-                </ul>
-              :
-                <ul className={ style['p-navigation__links--right']} role="menu">
-                  <li className={ style['p-navigation__link'] } role="menuitem">
-                    <a href="/auth/authenticate">Sign in with GitHub</a>
-                  </li>
-                </ul>
-            }
-          </nav>
+          <a href="#navigation" className={ style['p-navigation__toggle--open'] } title="menu">Menu</a>
+          <a href="#navigation-closed" className={ style['p-navigation__toggle--close'] } title="close menu">Close menu</a>
         </div>
+        <nav className={ style['p-navigation__nav'] } role="menubar">
+          <ul className={ style['p-navigation__links']} role="menu">
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href="https://snapcraft.io/store/">Store</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href="https://snapcraft.io/blog/">Blog</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a className={ style['is-selected'] } href="https://snapcraft.io/build">Build</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a href="https://docs.snapcraft.io">Docs</a>
+            </li>
+            <li className={ style['p-navigation__link'] } role="menuitem">
+              <a className={ style['p-link--external'] } href="https://forum.snapcraft.io/categories">Forum</a>
+            </li>
+          </ul>
+          { authenticated
+            ?
+              <ul className={ style['p-navigation__links--right']} role="menu">
+                <li className={ style['p-navigation__link']} role="menuitem">
+                  <a className={ style['p-dropdown__toggle']} aria-controls="account-menu" aria-expanded={ this.state.showUserDropdown }
+                    onClick={ this.onDropdownClick.bind(this)}>
+                    {user.name || user.login}<IconChevron/>
+                  </a>
+                  <ul className={ style['p-dropdown__menu']} id="account-menu" aria-hidden={ !this.state.showUserDropdown }>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href="https://snapcraft.io/account/snaps">My published snaps</a>
+                    </li>
+                    <li className={ style['p-navigation__link']} role="menuitem">
+                      <a href="/auth/authenticate">Build with GitHub</a>
```

Comment:
This link is quite confusing in BSI context.

I already am in BSI (possibly via this link), so maybe at least it could be marked as active?

Reply:
I'm wondering if there's a better title then "Build with Github", but I've updated the link as suggested.